server/routes/router.js

```diff
@@ -0,0 +1,65 @@
+import express from "express";
+const router = express.Router();
+import Article from "../models/blogmodules.js";
+
+router.get("/", async (req, res) => {
+	try {
+		const blog = await Article.find();
+		res.status(200).json(blog);
+	} catch (error) {
+		res.send({ error: "Article doesn't exist!" });
+	}
+});
+
+router.get("/:id", async (req, res) => {
+	try {
+		const blog = await Article.findOne({ _id: req.params.id });
+		res.status(200).json(blog);
+	} catch (error) {
+		res.send({ error: "Article does not exist" });
+	}
+});
+
+router.post("/", async (req, res) => {
+	try {
+		const blog = new Article({
+			title: req.body.title,
+			description: req.body.description,
+			image: req.body.image,
+		});
+		const blogs = await blog.save();
+		res.status(200).json(blogs);
```

Comment:
Looks good but i would like responses like this 

`res.status(200).json({"status":"success", "message":"all posts", "data":{"posts":posts}})`
`res.status(404).json({ "status":"success", "message":"no post found"})`

refer to resource i shared on the channel. 

we need to work on your file/folder structure. but it's okay for now
modify those responses and you should be fine. keep it up!